helpers/mapLabelNameService.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2024 (original work) Open Assessment Technologies SA ;
+ */
+
+ namespace oat\tao\helpers;
+ use oat\tao\helpers\Layout;
+
+ class MapLabelNameService
+{
+
+    private const ITEM = 'Item';
+    private const MEDIA = 'Media';
+    private const DELIVERY = 'Delivery';
+    private const ASSETS = 'Assets';
+
+    // New terms for isSolarDesignEnabled FF
+    private $mapLabelNames = [
+        self::ITEM => 'Item',
+        self::MEDIA => 'Asset',
+        self::DELIVERY => 'Delivery',
+        self::ASSETS => 'Asset'
+    ];
+
+    /**
+    * @param $labelName
+    * @return string
+    */
+    public function mapLabelName(string $labelName): string
+    {
+        $mapName = $labelName;
+        if (Layout::isSolarDesignEnabled()) {
+            $mapName = __(self::$mapLabelNames[$labelName]);
+        }
+        return $mapName;
+    }
```

Comment:
We can add the is solar design enabled as a parameter, so there is no need to have the hidden dependency Layout::isSolarDesignEnabled() it can be injected from outside => easier to test, easier to mock in tests.

```suggestion
    public static function mapLabelName(string $labelName, bool $isSolarDesignEnabled): string
    {
        if ($isSolarDesignEnabled && array_key_exists($labelName, self::$mapLabelNames)) {
            return self::$mapLabelNames[$labelName];
        }

        return $labelName;
    }
```